models/junos/security_policies_global/junos_security_policies_global.yml

```diff
@@ -0,0 +1,160 @@
+---
```

Comment:
we need to have fields required for resource module builder
https://github.com/ansible-network/resource_module_models/blob/f013e00f8bf0e48cbefff6853c35a98b0b2e5090/models/junos/ntp_global/junos_ntp_global.yaml?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L1